diff --git a/features/lomiri_common.prf b/features/lomiri_common.prf
index 159af6e01..6464beec6 100644
--- a/features/lomiri_common.prf
+++ b/features/lomiri_common.prf
@@ -13,20 +13,6 @@
 # order to workaround these issues, we unset these variables and set the
 # different flags apart from -g and -O2 in the right qmake variables. This
 # allows one to keep compiling with the Debian flags but in a more flexible way.
-!qt_submodule_build:!debian_build {
-    gcc:!clang {
-        QMAKE_CFLAGS = -fstack-protector
-        QMAKE_CXXFLAGS = -fstack-protector
-        QMAKE_CXXFLAGS_WARN_ON += -Wdate-time -Wformat -Werror=format-security
-        QMAKE_LFLAGS_SHLIB += -Wl,-z,relro
-        DEFINES += _FORTIFY_SOURCE=2
-        unix:!contains(QT_CONFIG, reduce_relocations): CONFIG += bsymbolic_functions
-    } else {
-        QMAKE_CFLAGS =
-        QMAKE_CXXFLAGS =
-    }
-    QMAKE_LFLAGS =
-}
 
 # Warn about overriding virtual functions not marked with the override keyword.
 gcc:!clang {
